code/src/components/GenreForm.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+export const GenreForm = ({title, genre, setGenre}) => {
+
+    return (
+        <article>
+        <h1>{title}</h1>
+        <select className="selectBox"
+        required
+        onChange={event => setGenre(event.target.value)}
+        value={genre}>
+        <option value="">Choose your genre:</option>
```

Comment:
I'm guessing you don't want this as an option
```suggestion
        <option value="" disabled>Choose your genre:</option>
```